src/org/zaproxy/zap/extension/websocket/manualsend/WebSocketPanelSender.java

```diff
@@ -44,10 +45,15 @@
     private static final Logger logger = Logger.getLogger(WebSocketPanelSender.class);
 
 	private Map<Integer, WebSocketProxy> connectedProxies;
-
+	private Map<Integer, WebSocketProxy> closedProxies;
+	private WebSocketReopenSend webSocketReopenSend;
     public WebSocketPanelSender() {
     	connectedProxies = new HashMap<>();
+    	closedProxies= new HashMap<>();
+    	this.webSocketReopenSend = webSocketReopenSend;

```

Comment:
Is this necessary? There isn't anything being passed into the constructor to change the value?